tedana/viz.py

```diff
@@ -245,6 +245,41 @@ def write_kappa_scatter(comptable, out_dir):
     plt.close()
 
 
+def write_kappa_scree(comptable, out_dir):
+    """
+    Creates a scree plot sorted by kappa, showing the values of the kappa and
+    rho metrics as well as the variance explained.
+
+    Parameters
+    ----------
+    comptable : (C x X) :obj:`pandas.DataFrame`
+        Component metric table. One row for each component, with a column for
+        each metric. Requires at least four columns: "classification",
+        "kappa", "rho", and "variance explained".
+    out_dir : :obj:`str`
+        Figures folder within output directory
+
+    """
+
+    fig, ax1 = plt.subplots(figsize=(10, 9))
+
+    ax1.plot(comptable.index.to_numpy(), comptable['variance explained'].to_numpy(),
+             'k-', alpha=0.5, linewidth=2, label='Variance')
+    ax1.set_ylabel('Variance Explained', fontsize=15)
+    ax2 = ax1.twinx()
+
+    ax2.plot(comptable.index.to_numpy(), comptable.kappa.to_numpy(),
+             'b-', linewidth=2, label='Kappa')
+    ax2.plot(comptable.index.to_numpy(), comptable.rho.to_numpy(),
+             'r-', linewidth=2, label='Rho')
+    ax2.set_title('Kappa/Rho Metrics', fontsize=28)
+    ax1.set_xlabel('Component Number', fontsize=15)
+    ax2.set_ylabel('Metric Value', fontsize=15)
+    fig.legend(loc='upper right', bbox_to_anchor=(0.82, 0.78))
+    screefig_title = os.path.join(out_dir, 'Kappa_Rho_Scree_plot.png')
+    plt.savefig(screefig_title)
```

Comment:
Could you use `fit.savefig` instead? Mostly a personal preference, but it makes it more explicit.

Reply:
seems reasonable to me